grammars/apache.cson

```diff
@@ -250,6 +250,23 @@
     ]
   }
   {
+    'begin': '\''
```

Comment:
I'm thinking you can get away with not escaping this by wrapping it in double quotes instead. I.e. `'begin': "'"`